[ADD] accounting: add Hong Kong localisations (#3657)
odoo/documentation: content/applications/finance/fiscal_localizations/hong_kong.rst

get the features of the HK localization.

.. image:: hong_kong/hk_01.png
   :align: center

Add FPS QR Codes to invoices
============================

Activate QR Code feature
------------------------

Go to :menuselection:`Accounting --> Configuration --> Settings`
and activate the **QR Code** feature.

.. image:: hong_kong/hk_02.png
   :align: center

Configure your FPS bank account
-------------------------------

Go to :menuselection:`Accounting --> Configuration --> Bank Accounts`,
open your FPS bank account and fill in the **FPS Type** and
**FPS ID/Phone Number/Email Address** fields.

.. image:: hong_kong/hk_03.png
   :align: center

Configure your Bank Account's journal
-------------------------------------

Go to :menuselection:`Accounting --> Configuration --> Journals`, open your
bank journal, then fill out the Account Number and Bank under the Journal
Entries tab.

.. image:: hong_kong/hk_04.png
   :align: center

Issue Invoices with FPS QR Codes
--------------------------------

To enable the FPS QR Code on your invoices, go to :menuselection:`Accounting
--> Customers --> Invoices`, and create a new invoice.

> Open the *Other Info* tab and select the *Payment QR-code* option to

Reviewer: Use the guilabel markup to identify any text of the interactive user interface (e.g., button labels, view titles, field names, lists items, …).

=========
Hong Kong
=========

Configuration
=============

:ref:`Install <general/install>` the :guilabel:`Hong Kong - Accounting` module to
get the features of the HK localization.

> .. image:: hong_kong/hk_01.png
>    :align: center

Reviewer: ```suggestion
.. image:: hong_kong/hk-module.png
   :align: center
   :alt: "Hong Kong - Accounting" localization module marked as "installed"
```

Add FPS QR Codes to invoices
============================

Activate QR Code feature
------------------------

Go to :menuselection:`Accounting --> Configuration --> Settings`
and activate the **QR Code** feature.

.. image:: hong_kong/hk_02.png
   :align: center

Configure your FPS bank account
-------------------------------

Go to :menuselection:`Contacts --> Configuration --> Bank Accounts`,

> select a Account Holder with Hong Kong as country, open your FPS bank

Reviewer: ```suggestion
select an account holder with Hong Kong as country, open your FPS bank
```
I no longer see where to "open your FPS bank account and fill in the "**FPS Type** and **FPS ID/Phone Number/Email Address** fields. Is there any configuration needed to be able to see it? It worked on the runbot you sent but not when I try to reproduce it in another runbot.

Author: The feature is not yet deployed to master therefore is not existed yet on other Runbot.

account and fill in the **FPS Type** and **FPS ID/Phone Number/Email
Address** fields.

.. image:: hong_kong/hk_03.png
   :align: center

Configure your Bank Account's journal
-------------------------------------

Go to :menuselection:`Accounting --> Configuration --> Journals`, open your
bank journal, then fill out the Account Number and Bank under the Journal
Entries tab.

.. image:: hong_kong/hk_04.png

>    :align: center

Reviewer: Suggestion to only add screenshots of the relevant area, rather than keeping wide areas with arrows or squares to show what we want to illustrate.

Issue Invoices with FPS QR Codes
--------------------------------

To enable the FPS QR Code on your invoices, go to :menuselection:`Accounting
--> Customers --> Invoices`, and create a new invoice.

Open the *Other Info* tab and select the *Payment QR-code* option to
*FPS Transfer QR*.

.. image:: hong_kong/hk_05.png
   :align: center

Also check the *Recipient Bank* field and make sure that the account indicated
is the one you want to use to receive your customer's payment.

.. image:: hong_kong/hk_06.png

>    :align: center

Reviewer: I wouldn't add this image as such. If you want to illustrate how the QR code displays on the invoice, I would  add a short explanation and then the image.

=========
Hong Kong
=========

Configuration
=============

:ref:`Install <general/install>` the :guilabel:`Hong Kong - Accounting` module to get the features
of the HK localization.

.. image:: hong_kong/hk-module.png
   :align: center
   :alt: "Hong Kong - Accounting" localization module marked as "installed"

Add FPS QR Codes to invoices
============================

:abbr:`FPS (Faster Payment System)` is a payment service platform that allows customers to make
instant domestic payments to individuals and merchants in Hong Kong dollar or Renminbi via online
and mobile banking.

Activate QR Code feature
------------------------

Go to :menuselection:`Accounting --> Configuration --> Settings` and then in the :guilabel:`Customer
Payments` section to activate the :guilabel:`QR Codes` feature.

.. image:: hong_kong/hk-qr-code-setting.png
   :align: center
   :alt: "QR Codes" feature activated

FPS bank account configuration
------------------------------

Go to :menuselection:`Contacts --> Configuration --> Bank Accounts`, select an account Holder with
Hong Kong as country, open your FPS bank account and fill in the :guilabel:`FPS Type` and

> :guilabel:`FPS ID/Phone Number/Email Address` fields.

Author: Yes the module is not on 16.0 runbot yet. If you need to test it, I will rebuild a runbot on my branch for your testing. Please feel free to let me know if you need it.

.. image:: hong_kong/hk-fps-bank-setting.png
   :align: center
   :alt: FPS bank account configuration

.. seealso::
   :doc:`../accounting/bank/setup/bank_accounts`

Bank Account's journal configuration
------------------------------------

Go to :menuselection:`Accounting --> Configuration --> Journals`, open your bank journal, then fill
out the Account Number and Bank under the Journal Entries tab.

.. image:: hong_kong/hk-bank-account-journal-setting.png
   :align: center
   :alt: Bank Account's journal configuration

Issue Invoices with FPS QR Codes
--------------------------------

To enable the FPS QR Code on your invoices, go to :menuselection:`Accounting --> Customers -->
Invoices`, and click :guilabel:`New` to create a new invoice.

Open the :guilabel:`Other Info` tab and select the  :guilabel:`Payment QR-code` option to *EMV
Merchant-Presented QR-code*.

.. image:: hong_kong/hk-qr-code-invoice-setting.png
   :align: center
   :alt: Select EMV Merchant-Presented QR-code option

> Make sure that the :guilabel:`Recipient Bank` is the one you want to use to receive your customer's
> payment as Odoo uses this field to generate the FPS QR Code.

Reviewer 2: ```suggestion
Ensure that the :guilabel:`Recipient Bank` is the one you configured, as Odoo uses this field to generate the FPS QR code.
```
I would rephrase to clarify that you must select the one you just configured, not something else. If you need to specify "the one you want to use to receive customers' payments", move that info to the "FPS bank account configuration" heading.
